README.md

```diff
@@ -132,6 +132,25 @@ Before submitting any changes to the repository, make sure to run `grunt test`
 to make sure it passes all unit tests. Failing tests are enough to immediately
 reject submissions. :)
 
+### uProxy for Android
+
+uProxy can now be built as an Android App, using the Cordova Chrome Apps (CCA)
+tool, also known as the [Chrome Apps for Mobile Toolchain](https://github.com/MobileChromeApps/mobile-chrome-apps).
+After following the steps under "Building uProxy from source",
+ 1. Install [Android Studio](https://developer.android.com/sdk/index.html).
```

Comment:
I think it's worth clarifying that you really just need the Android SDK and `adb` and that Android Studio is the *easiest* way to get these (although Android Studio doesn't seem to include `adb` on Linux, see note below).